dashboards/k8s-views-global.json

```diff
@@ -153,11 +152,25 @@
           },
           "editorMode": "code",
           "exemplar": true,
-          "expr": "avg(sum by (instance, cpu) (rate(node_cpu_seconds_total{mode!~\"idle|iowait|steal\", cluster=\"$cluster\"}[$__rate_interval])))",
+          "expr": "avg(sum by (cpu) (rate(node_cpu_seconds_total{mode!=\"idle\", cluster=\"$cluster\"}[$__rate_interval])))",
           "interval": "",
-          "legendFormat": "Real",
+          "legendFormat": "Real Linux",
```

Comment:
Yes, there is a transformations which merge both Real queries together.

Otherwise, the prometheus query for both windows and linux is quite complex and hard to understand. I choice to use dedicated queries for windows and linux to keep the it more simple for other users. (See line 201+)